packages/ui/src/components/RecentOrderCarousel/RecentOrderCarousel.tsx

```diff
@@ -0,0 +1,45 @@
+'use client'
+import { RecentOrderItem, RecentOrderItemProps } from '../RecentOrderItem'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import 'swiper/swiper-bundle.css'
+import '../../styles/global.css'
```

Comment:
I believe you no need to import the global styles here. It should work good with out this import.